### DISCUSSION TITLE
Modern Search - Localization Auto owstaxidmetadataalltagsinfo

### DISCUSSION THREAD

**Discussion by Lotsamocha01** [microsoft-search/pnp-modern-search#2996] [Q&A]
I have a PnP Search Results webpart (v4.8) configured with Microsoft Search as the source with a List template. The Tags metadata was not showing for the documents until I changed the layout slot for Tags to 'resource.listitem.fields.owstaxidmetadataalltagsinfo'. I'm wondering if there is a way to enable Localization or Auto so the tags are showing in a friendly format instead of the full taxonomy path.
I also noticed that the Date doesn't show unless I change the layout slot from 'created' to 'resource.createdDateTime' and the Author slot was missing and needed to be added with 'resource.createdBy.user.displayName'.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/114703957/1f25a30e-9381-47b0-a93f-43bff1d1b933)

**Comment by wobba** [comment]
Localization with the Graph provider is not supported today.
